.github/workflows/stable-compilation.yml

```diff
@@ -52,12 +52,13 @@ jobs:
 
       - name: Compile
         run: |
+          VER="(GA, `date +%Y-%m-%d`)"
```

Comment:
Looking good! Is there a simple way to override this date value later to support reproducible builds? Or an alternate, more deterministic way by getting the tag/commit date from git, for example (the release date could be different, however).

Reply:
Had a look at this before. Currently the date is just the default string if not provided. Maintainers can just overwrite it to empty for example. 

Some time in the future we can add support for `SOURCE_DATE_EPOCH`, which adds up on that. Then `PLAYER_DATE` could get introduced to implement this.

Reply:
Using an autoconf archive macro solves this for now.